File: CS/E911.Module/DatabaseUpdate/Updater.cs
using System;
using DevExpress.ExpressApp;
using DevExpress.ExpressApp.Updating;
using DevExpress.Persistent.BaseImpl;

namespace E911.Module.DatabaseUpdate {
    public class Updater : ModuleUpdater {
        public Updater(IObjectSpace objectSpace, Version currentDBVersion) : base(objectSpace, currentDBVersion) { }
        public override void UpdateDatabaseAfterUpdateSchema() {
            base.UpdateDatabaseAfterUpdateSchema();
            for (int i = 0; i < 10; i++) {
                Task theObject = ObjectSpace.CreateObject<Task>();
                theObject.Subject = String.Format("Task{0}", i);
            }
            ObjectSpace.CommitChanges();
        }
    }
}

File: VB/VB/E911.Module/DatabaseUpdate/Updater.vb
Imports DevExpress.ExpressApp
Imports DevExpress.ExpressApp.Updating
Imports DevExpress.Persistent.BaseImpl

Namespace E911.Module.DatabaseUpdate
	Public Class Updater
		Inherits ModuleUpdater
		Public Sub New(ByVal objectSpace As IObjectSpace, ByVal currentDBVersion As Version)
			MyBase.New(objectSpace, currentDBVersion)
		End Sub
		Public Overrides Sub UpdateDatabaseAfterUpdateSchema()
			MyBase.UpdateDatabaseAfterUpdateSchema()
			For i As Integer = 0 To 9
				Dim theObject As Task = ObjectSpace.CreateObject(Of Task)()
				theObject.Subject = String.Format("Task{0}", i)
			Next i
			ObjectSpace.CommitChanges()
		End Sub
	End Class
End Namespace
